diff --git a/launcher/launcher.pro b/launcher/launcher.pro
index 1f2bf6c..fbd06ad 100644
--- a/launcher/launcher.pro
+++ b/launcher/launcher.pro
@@ -59,7 +59,6 @@ RESOURCES += \
 # For QArchive
 QT += concurrent
 
-LIBS += ../libs/QArchive/libQArchive.a -larchive -lbz2 -llzma -llz4 -lz -lzstd -lnettle -lexpat -lxml2
-win32:LIBS += -lbcrypt -liconv
+LIBS += ../libs/QArchive/release/libQArchive.a -larchive -lnettle -lxml2
 
 INCLUDEPATH += ../libs/QArchive ../libs/QArchive/include
